src/TimeControl.cpp

```diff
@@ -92,6 +92,9 @@ void TimeControl::stop(int color) {
 
     assert(elapsed_centis >= 0);
 
+    // because of GUI/network lag, cfg_lagbuffer_cs centisecond is lost on each move
+    elapsed_centis += cfg_lagbuffer_cs;
```

Comment:
I'm not clear whether we should pull this. It indicates an interface that is completely broken.